src/main/webapp/app/pages/curation/review/ReviewPage.tsx

```diff
@@ -206,6 +206,7 @@ const ReviewPage: React.FunctionComponent<IReviewPageProps> = (props: IReviewPag
           </Row>
         </>
       )}
+      {(isAcceptingAll || isAccepting) && <LoadingIndicator key={'curation-page-loading'} size={LoaderSize.LARGE} center isLoading />}
```

Comment:
This is new to me. The loader is on top of the review page. What's your thought on this?

Reply:
Yes it's at the top. I have no real preference on this. You have a better eye for UI than I do.

Reply:
We could hide all review content when accepting all, and show a loader on the white background. Could you try that? Or without.